### PR TITLE
Docs: Update Prop Table to support Markdown for descriptions

### PR DESCRIPTION
**Pull request by ayeshakmaz**
Change prop table descriptions to use Markdown (allows for links)

Before: 
Screen Shot 2021-02-10 at 6 05 00 PM

After:
Screen Shot 2021-02-10 at 1 20 59 PM
Screen Shot 2021-02-10 at 6 04 49 PM

### CODE REVIEW

**Comment by AlbertCarreras** [comment]
Can you add a screenshot to the description to see the before/after?

File: docs/src/components/PropTable.js [hunk 1]
```diff
@@ -201,7 +200,11 @@ export default function PropTable({
                             )}
                           </Td>
                           <Td colspan={1} color="gray">
-                            {Array.isArray(description) ? Description(description) : description}
+                            {Array.isArray(description) ? (
+                              Description(description)
```
Comment (AlbertCarreras) [inline docs/src/components/PropTable.js:204]: ONe all docs migrated, Arrays wont be needed right?

File: docs/src/components/Markdown.js [hunk 2]
```diff
@@ -65,7 +66,7 @@ const formatComponentName = (listitem) => {
   return listitem;
 };
 
-export default function Markdown({ text, type }: Props): Node {
+export default function Markdown({ color, text, type }: Props): Node {
```
Comment (AlbertCarreras) [inline docs/src/components/Markdown.js:69]: just a thought.. would `textColor` be more explicit? I wasn't sure if color was background etc

File: docs/src/TapArea.doc.js [hunk 3]
```diff
@@ -203,8 +203,8 @@ card(
         defaultValue: 'button',
         description: [
           `Select a TapArea variant:`,
-          `- 'button': Use for TapArea to act like buttons. The TapArea is rendered as a '<div>'.`,
-          `- 'link': Use for TapArea to act like links. The button is rendered as an '<a>'.`,
+          `- 'button': Use for TapArea to act like buttons. The TapArea is rendered as a \`<div>\`.`,
+          `- 'link': Use for TapArea to act like links. The button is rendered as an \`<a>\`.`,
```
Comment (AlbertCarreras) [inline docs/src/TapArea.doc.js:207]: thanks for correcting all these one.

**Comment by AlbertCarreras** [comment]
I'd r+ but see all the red lights

File: docs/src/components/Markdown.js (hunk 2 above)
Comment (ayeshakmaz) [reply docs/src/components/Markdown.js:69]: mmm good call, will update

File: docs/src/components/PropTable.js (hunk 1 above)
Comment (ayeshakmaz) [reply docs/src/components/PropTable.js:204]: yup! we can eventually remove that

**Review by AlbertCarreras — APPROVED** [review]

**Pull request merged**